jerry-core/debugger/debugger.h

```diff
@@ -341,9 +340,8 @@ typedef struct
  */
 typedef struct
 {
-  jerry_debugger_send_header_t header; /**< message header */
   uint8_t type; /**< type of the message */
-  jerry_debugger_frame_t frames[JERRY_DEBUGGER_SEND_MAX (jerry_debugger_frame_t)]; /**< frames */
+  jerry_debugger_frame_t frames[1]; /**< frames */
```

Comment:
Why `[1]` and not `[]` (also in the other structs)?
Aren't these variable sized arrays?

Reply:
Isn't that part of C99 (the README claims that the project uses this standard)?

https://en.wikipedia.org/wiki/Flexible_array_member

> In previous standards of the C language, it was common to declare a zero-sized array member instead of a flexible array member. The GCC compiler explicitly accepts zero-sized arrays for such purposes.

Maybe use that instead?

Reply:
Which one? Coverity has no problem with it:
https://travis-ci.org/jerryscript-project/jerryscript/jobs/361615057

Reply:
I just realized that the definition is wrong (non-packed form), and it only works because it is never used. Perhaps we could replace the second member with a comment.

Reply:
Yeah, that makes sense to me.

Reply:
I finally chose `[]`. I hope it is really part of C99.